day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -273,14 +273,18 @@ Before restoring etcd on a static pod:
 * `etcdctl` binaries must be available or, in containerized installations,
 the `rhel7/etcd` container must be available.
 +
-You can obtain etcd by running the following commands:
+You can install the `etcdctl` binary with the etcd package by running the following commands:
 +
 ----
-$ git clone https://github.com/coreos/etcd.git
-$ cd etcd
-$ ./build
+# yum install etcd
 ----
 
+The package also installs the systemd service. Disable and mask the service as it shouldn't run as a systemd service when etcd runs in static pod.
```

Comment:
yeah, but we won't to make sure that the service is not started manually by mistake. it is not required, but if not masked it can be started. The disable is to make sure it will not start if the system is rebooted. It is more for safety.